pyanaconda/modules/localization/localed.py

```diff
@@ -52,7 +52,6 @@ def __init__(self):
 
         self._localed_proxy = LOCALED.get_proxy()
 
-    @property
     def layouts_variants(self):
```

Comment:
I think we should rename it as well, to reflect that it's a getter.  `def get_layouts_variants`
The `Code conventions` do not mention such suggestion, I still think it makes sense.

Reply:
Good point.